docparse/output_walkthrough.mdx

```diff
@@ -0,0 +1,176 @@
+
+<Frame>
+  <img src="../images/59_page2.png" />
+</Frame>
+
+## Getting Parsed Document
+
+### Making a Call to DocParse
+
+```python text_demo.py
+with open(file_name, 'rb') as file:
+  ## Make a call to the partitioning service and set extract_images to true.
+  partitioned_file = partition_file(file, aryn_api_key, extract_images=True, extract_table_structure=True, use_ocr=False, selected_pages=[2])
+```
+We use the `partition_file` function from `aryn_sdk.partition` to extract text and images from the document. The `aryn_api_key` is your API key from Aryn. Set `extract_images` and `extract_table_structure` to `True` to extract images and tables. Set `use_ocr` to `False` to avoid OCR and use embedded text. We use `selected_pages=[2]` to focus on page 2.
+
+### Viewing the JSON Output
+
+```json output.json
+{'status': ['Incremental status will be shown here during execution.',
+  "Until you get a line that matches '  ]\n', you can convert the partial",
+  'output to a json document by appending \'""]}\' to the partial output.',
+  '',
+  'T+   0.00: Server version managed-service-22-model-1-4 Model version 1.4',
+  'T+   0.00: Received request with aryn_call_id=REDACTED',
+  'T+   0.00: Waiting for scheduling',
+  'T+   0.01: Preprocessing document',
+  'T+   0.01: Done preprocessing document',
+  'T+   0.51: Completed work on page 2',
+  ''],
+ 'elements': [{'type': 'Image',
+   'bbox': [0.0873676793715533,
+    0.08911955399946733,
+    0.8548237161075367,
+    0.5357984508167614],
+   'properties': {'score': 0.873895525932312,
+    'image_size': [1324, 1003],
+    'image_mode': 'RGB',
+    'image_format': None,
+    '_element_index': 1,
+    'page_number': 2},
+   'binary_representation': ...
+ }
+```
+
+Above, you can see the JSON output from the call to Aryn DocParse. The output is a JSON object with a `status` field that shows the status of the call and an `elements` field that contains a list of elements extracted from the document. Each element has a `type`, `bbox`, `properties`, and `text_representation` field. The `type` field indicates the type of the element (e.g. text, image, table, etc.), the `bbox` field contains the coordinates of the bounding box around the element, the `properties` field contains additional information about the element (e.g. confidence score, page number, etc.), and the `text_representation` field contains the text content of the element. The call ID has been redacted for privacy reasons.
+
+## Examining Individual Elements
+
+### Extracting an Image from the Document
+
+```json image.json
+
+{ 
+  type': 'Image',
+ 'bbox': [0.0873676793715533,
+  0.08911955399946733,
+  0.8548237161075367,
+  0.5357984508167614],
+ 'properties': {'score': 0.873895525932312,
+  'image_size': [1324, 1003],
+  'image_mode': 'RGB',
+  'image_format': None,
```

Comment:
are there other image modes or formats? Do we need docs for those?

Reply:
Also not something that should be in the output, good catch.